.github/workflows/performance.yml

```diff
@@ -188,8 +189,11 @@ jobs:
           fi
           git reset --hard $TARGET_SHA
 
-      - name: Check desired node version
-        run: nvm install
+      - name: Set up Node.js
```

Comment:
We later call `git reset --hard $GITHUB_SHA` in the `Reset to original commit` step. Before that, the same is needed.

Reply:
Should be resolved now!